clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/CreateServerGroupWithAppAutoScalingSpec.java

```diff
@@ -0,0 +1,248 @@
+                  DescribeTargetGroupsRequest request =
+                      (DescribeTargetGroupsRequest) invocation.getArguments()[0];
+                  String testArn =
+                      "arn:aws:elasticloadbalancing:::targetgroup/"
+                          + request.getNames().get(0)
+                          + "/76tgredfc";
+                  TargetGroup testTg = new TargetGroup().withTargetGroupArn(testArn);
+
+                  return new DescribeTargetGroupsResult().withTargetGroups(testTg);
+                });
+
+    when(mockAWSApplicationAutoScalingClient.describeScalableTargets(
+            any(DescribeScalableTargetsRequest.class)))
+        .thenAnswer(
+            (Answer<DescribeScalableTargetsResult>)
+                invocation -> {
+                  ScalableTarget scalableTarget =
+                      new ScalableTarget()
+                          .withMaxCapacity(1)
+                          .withMinCapacity(1)
+                          .withResourceId("service/default/sample-webapp");
+                  return new DescribeScalableTargetsResult().withScalableTargets(scalableTarget);
+                });
+
+    when(mockAWSApplicationAutoScalingClient.describeScalingPolicies(
+            any(DescribeScalingPoliciesRequest.class)))
+        .thenAnswer(
+            (Answer<DescribeScalingPoliciesResult>)
+                invocation -> {
+                  Alarm alarm =
+                      new Alarm()
+                          .withAlarmARN("arn:aws:cloudwatch:us-east-1:123456789012:alarm:testAlarm")
+                          .withAlarmName("testAlarm");
+                  ScalingPolicy scalablePolicy =
+                      new ScalingPolicy()
+                          .withResourceId("service/default/sample-webapp")
+                          .withPolicyName("ecsTestPolicy")
+                          .withPolicyARN(
+                              "arn:aws:autoscaling:us-west-2:012345678910:scalingPolicy:6d8972f3-efc8-437c-92d1-6270f29a66e7:resource/ecs/service/default/web-app:policyName/web-app-cpu-gt-75")
+                          .withAlarms(Arrays.asList(alarm));
+                  return new DescribeScalingPoliciesResult()
+                      .withScalingPolicies(Arrays.asList(scalablePolicy));
+                });
+
+    when(mockAWSApplicationAutoScalingClient.putScalingPolicy(any(PutScalingPolicyRequest.class)))
+        .thenReturn(
+            new PutScalingPolicyResult()
+                .withPolicyARN(
+                    "arn:aws:autoscaling:us-west-2:012345678910:scalingPolicy:6d8972f3-efc8-437c-92d1-6270f29a66e7:resource/ecs/service/default/web-app:policyName/web-app-cpu-gt-75"));
+
+    when(mockAmazonCloudWatchClient.describeAlarms(any(DescribeAlarmsRequest.class)))
+        .thenReturn(
+            new DescribeAlarmsResult()
+                .withMetricAlarms(
+                    Arrays.asList(new MetricAlarm().withAlarmName("TargetTracking-testAlarm"))));
```

Comment:
this is the only place we refer to the test alarm as "TargetTracking-testAlarm" (everywhere else it's just "testAlarm"). I know we're not asserting on this mocked return value, but it was confusing to see assertions on the `DescribeAlarmsResults` args below succeeding with a different name. Can we please use one or the other everywhere?

Reply:
@allisaurus I have made sure to keep the name consistent i.e. "testAlarm" everywhere.